analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -63,12 +63,9 @@ import org.ossreviewtoolkit.utils.collectMessagesAsString
 import org.ossreviewtoolkit.utils.log
 import org.ossreviewtoolkit.utils.logOnce
 
-// See https://docs.microsoft.com/en-us/nuget/api/overview.
-private const val SERVICE_INDEX_URL = "https://api.nuget.org/v3/index.json"
-
 private const val REGISTRATIONS_BASE_URL_TYPE = "RegistrationsBaseUrl/3.6.0"
 
-class NuGetSupport {
+class NuGetSupport(vararg serviceIndexUrls: String = arrayOf("https://api.nuget.org/v3/index.json")) {
```

Comment:
Please keep this as a constant to denote it's special standing as *the* service index URL.

Reply:
And you can omit the surrounding `arrayOf()` here.

Reply:
Moved the URL to a constant. But `arrayOf()` is required.